.changeset/five-roses-drum.md

```diff
@@ -0,0 +1,6 @@
+---
+'@microsoft/atlas-css': minor
+'@microsoft/atlas-site': minor
+---
+
+Adding new colors to the palette. Adding the use case example to the site.
```

Comment:
nit: helpful if changesets are specific. maybe we could mention sand color and blue black to palette, as well as new theme color (body-background-accent)

Reply:
Updated